src/SearchParameters/RefinementList.js

```diff
@@ -74,8 +74,8 @@ var lib = {
    * @param {string} value the value of the refinement
    * @return {RefinementList} a new and updated list
```

Comment:
In the SearchParameters:

 - toggleRefinement: should stay, be deprecated, and should be an alias to the new toggleFacetRefinement which will have the same implementation
 - toggleFacetRefinement (the old method): should be renamed to toggleConjunctiveFacetRefinement

In the SearchParameters:

 - toggleRefinement: should be renamed toggleFacetRefinement, and we should still provide a deprecated alias from toggleRefinement to toggleFacetRefinement

There are no changes needed in RefinementList.

Let me know if you have questions on all that :)